README.md

```diff
@@ -37,22 +52,10 @@ You will also need to install dependencies for nltk if you do not already have t
 python -m nltk.downloader -d /usr/share/nltk_data perluniprops nonbreaking_prefixes punkt
 ```
 
-## Submodules
 
-This project uses [git submodules](https://blog.github.com/2016-02-01-working-with-submodules/) to manage some dependencies on other research code, in particular for loading CoVe and the OpenAI transformer model. In order to make sure you get these repos when you download `jiant/`, add `--recursive` to your clone command:
+### Downloading data
 
-```sh
-git clone --recursive git@github.com:jsalt18-sentence-repl/jiant.git jiant
-```
-
-If you already cloned and just need to get the submodules, you can do:
-```sh
-git submodule update --init --recursive
-```
-
-## Downloading data
-
-The repo contains a convenience python script for downloading all [GLUE](https://www.nyu.edu/projects/bowman/glue.pdf) data and standard splits.
+The repo contains a convenience python script for downloading all [GLUE](https://gluebenchmark.com/) data and standard splits.
```

Comment:
small thing, but previously this script was in a git gist, and that was the main version I updated. It'd be nice to have it in a single place, either this repo or a gist, so we don't have to maintain two versions.